### DISCUSSION TITLE
Arduino IDE

### DISCUSSION THREAD

**Discussion by Mjgxy** [dawigit/picoclock#11] [Q&A]
Is it possible to code this using Arduino IDE or do you have to use minicom. because I cant get minicom to work for me.

**Comment by dawigit** [comment]
I don't now for sure - i wasn't able to just installing Arduino IDE and adding RP2040.
Raspberry Pi recommends VSCode.

Did you:

`sudo usermod -a -G dialout <username>`

add your user to the 'dialout' group?
HOW does minicom NOT work?
Error message?
Or is it just starting, hanging a bit, then quits?

**Comment by Mjgxy** [comment]
First off thank you for trying to use Arduino IDE with the RP2040, and
second of all I'm new to the coding community and have little experience.

Back to the point I wasn't able to install minicom on my computer, granted
it is an older computer running on the latest version of ubuntu I don't
necessarily if that has an effect on it or not but Ill try what to
suggested and get back to you on the results.
